plugins/CoreHome/Columns/Metrics/ConversionRate.php

```diff
@@ -43,8 +44,20 @@ public function format($value, Formatter $formatter)
 
     public function compute(Row $row)
     {
-        $nbVisitsConverted = $this->getMetric($row, 'nb_visits_converted');
         $nbVisits = $this->getMetric($row, 'nb_visits');
+        $nbVisitsConverted = $this->getMetric($row, 'nb_visits_converted');
+        if ($nbVisitsConverted === false) {
+            $goals = $this->getMetric($row, 'goals');
+
+            $nbVisitsConverted = 0;
+            foreach ($goals as $idGoal => $values) {
+                if ($idGoal === 0 || $idGoal === '0' || $idGoal == 'idgoal=0') {
```

Comment:
would this also need to be done for `goals = -1`?
@mattab maybe you can have a look at the PR some time I actually don't quite understand if the behaviour is expected.

Reply:
yeah, that makes sense don't want to count "conversions" for abandoned carts here